assets/sass/modules/_btns.scss

```diff
@@ -102,6 +102,29 @@
   }
 }
 
+.piv-button.link-button {
+  @extend .button-wide;
+  border-radius: $border-radius-default;
+  box-sizing: border-box;
+
+  display: block;
+  font-size: $font-size;
+  line-height: 50px;
+  margin-top: 15px;
+  text-align: center;
+
+  &:last-of-type {
```

Comment:
is the intention here to have a margin bottom 25px; only for the last button (be it default/piv/social) 

if so we should add it to a diff class that is common to all three. Otherwise there will be a margin bottom added for each button since they are all unique.

Reply:
It is a very rare case to have a piv button with social auth or custom buttons.. But the way it would render if all three types were present is:
1. social auth buttons
2. piv
3. custom buttons

1. & 3. can have multiple buttons of that type, hence last-of-type adds some extra margin at the bottom to indicate the end that type of button. I wrote the same for piv, though there will only ever be 1 piv button.. I could add this margin-bottom after the text-align above instead. Please let me know what you think